libs/dal/src/repositories/job/job.schema.ts

```diff
@@ -143,5 +134,80 @@ jobSchema.virtual('environment', {
   justOne: true,
 });
 
+/*
+ * This index was initially created to optimize:
+ * apps/api/src/app/events/usecases/add-job/add-delay-job.usecase.ts
+ */
```

Comment:
Leaving the explanations for the different index optimisations are very helpful. 🙌🏻

Reply:
I think that even better would be to provide here some query examples? Because this comment is already not accurate as @LetItRock moved this file :P 

Having some query examples here stay accurate even when code moves.